### ISSUE TITLE
offset in perspective

### ISSUE THREAD

**Issue by myneu** [Tsinghua-MARS-Lab/HDMapNet#53]
Hello, great work!

According to my understanding, we project coordinate from camera to ego and bev mask
to ego:
pix_coords = proj_mat @ cam_coords
to mask:
pix_coords[:, 0] += offset[0] / 2
pix_coords[:, 2] -= offset[1] / 8
pix_coords = torch.stack([pix_coords[:, 2], pix_coords[:, 0]], axis=1)

But, I am confused about -offset[1]/8, why not +offset[1]/2. Looking forward to your reply.

Thanks in advance!
